### DISCUSSION TITLE
@vue/cli 4.5.15  " [WDS] Disconnected! "

### DISCUSSION THREAD

**Discussion by patialo** [coder/code-server#4909] [Q&A]
I use **_the domain name_** to access code-server and install the initialized project through vuecli
after "npm run serve"
![e96e05f2fde612f9b083b587bd65ef1](https://user-images.githubusercontent.com/31113826/155696383-befacf7b-3d8b-4cef-9bb6-e2f0849d4950.png)

Use **_ip:8080_** to access the running of this project, but the console keeps prompting "[WDS] Disconnected!"
![efbda8d934bb12db5be69ac75b820ac](https://user-images.githubusercontent.com/31113826/155696311-7e0df50f-50f3-4ea4-894a-6101d8ff8cd7.png)

This is just an initialization project without any configuration
I tried many ways to fix it, but none worked(such as : configure vue.config.js-devserver-{host: 127.0.0.1}or{0.0.0.0} )
(When copying the whole project to my computer (including node_modules), after "npm install" on my vscode ,it doesn't produce this error, there are several packages changed)
![f6c1834c034329adb1faa2b5d8bcef6](https://user-images.githubusercontent.com/31113826/155696260-28d24ba4-66e5-4a6d-89e8-f0532a289a4d.png)

what should I check to fix this please？

**Comment by jsjoeio** [comment]
Take a look at our [docs here](https://coder.com/docs/code-server/latest/guide#proxying-to-a-vue-app) for proxying a Vue app. That might help.

**Reply by patialo** [reply]
I tried it but doesn't work for me

![3a60498cf5b02e3a2ee567308579703](https://user-images.githubusercontent.com/31113826/155825876-ac289389-062d-453c-96b4-7c9ffa8675bb.png)
![76666c22daf8ff466b92cdced5a759d](https://user-images.githubusercontent.com/31113826/155825889-dbe61329-972f-4de2-bc2b-7371bc3853c6.png)
![802b8194e3f380d61a6caa5d6b21530](https://user-images.githubusercontent.com/31113826/155825887-ff67bc70-0039-4109-a114-78dc3b51bc9a.png)

**Reply by patialo** [reply]
as what in the document， i tried these way to access but not work too:
`https://<my-domain-name>/absproxy/3454`
`http://<IP>:8080/absproxy/3454`

**Reply by patialo** [reply]
In addition,when I copy the program that works fine on vscode to the code-server, I get an error when I use "npm run serve" without "npm inastall"
![8445bc1b6f6c305cad7caa06a0a1c3a](https://user-images.githubusercontent.com/31113826/155826572-2af3ace5-fd18-449a-ab3d-e08dd6120846.png)

So why is the dependency installed using npm install on code-server different from vscode?(they have the same npm registry)

**Reply by jsjoeio** [reply]
That is very strange 🤔 I have no idea to be honest. I've run into issues in the past with Vite with HMR and similar things. I'm not sure what we can do to fix this 🤔

**Reply by patialo** [reply]
This problem was solved by running code-server in docker
